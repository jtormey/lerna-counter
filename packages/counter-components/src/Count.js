import React from 'react'

const Count = ({ value }) => (
  <h1>Counter: {value}</h1>
)

export default Count
