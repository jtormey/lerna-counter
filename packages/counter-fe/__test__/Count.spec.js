import React from 'react'
import Count from '../src/Count'
import renderer from 'react-test-renderer'

describe('Count', () => {
  it('should render a value', () => {
    let tree = renderer.create(
      <Count value={9} />
    )
    expect(tree.toJSON()).toMatchSnapshot()
  })
})
