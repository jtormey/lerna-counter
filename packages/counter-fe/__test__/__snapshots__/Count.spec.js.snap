// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Count should render a value 1`] = `
<h1>
  Counter: 
  9
</h1>
`;
